.github/workflows/build.yml

```diff
@@ -7,6 +7,7 @@ jobs:
   build:
     name: Build
     runs-on: ubuntu-latest
+    environment: test-pr
```

Comment:
Created an Github environment called `test-pr` to have different secrets pointing to dummy app so that fork PRs can access them (still need to test this though).